services/web/public/icons/index.js

```diff
@@ -61,4 +62,5 @@ export {
   AddIcon,
   PenIcon,
   FileUploadIcon,
+  GoogleIcon,
```

Comment:
let's use tabler's Google icon, we already have a library of icons on the web
https://tabler-icons.io/